documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2717,6 +2717,14 @@ Set the `snapshot.mode.custom.name` configuration property to the name provided
 For more information, see xref:connector-custom-snapshot[custom snapshotter SPI].
 endif::community[]
 
+ifdef::community[]
+|[[db2-property-snapshot-mode-custom-name]]<<db2-property-snapshot-mode-custom-name, `+snapshot.mode.custom.name+`>>
+|No default
+| When `snapshot.mode` is set as `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.Snapshotter' interface.
+The provided implementation is called after a connector restart to determine whether to perform a snapshot.
```

Comment:
```suggestion
| If `snapshot.mode` is set to `custom`, use this setting to specify the name of the custom class that implements the `name()` method that is defined in the 'io.debezium.spi.snapshot.Snapshotter' interface.
After a connector restart, {prodname} calls the specified custom implementation to determine whether to perform a snapshot.
```

Reply:
@roldanbob The value of this property is the value provided in the `name()` method of the custom implementation, not the name of the custom class.

Reply:
> @roldanbob The value of this property is the value provided in the `name()` method of the custom implementation, not the name of the custom class.

Ah! Ok, got it. Thanks! Updated comments. Is this better?